src/java/org/apache/cassandra/config/CassandraRelevantProperties.java

```diff
@@ -463,6 +463,7 @@ public enum CassandraRelevantProperties
     CUSTOM_KEYSPACES_FILTER_PROVIDER("cassandra.custom_keyspaces_filter_provider_class"),
 
     LWT_LOCKS_PER_THREAD("cassandra.lwt_locks_per_thread", "1024"),
+    LWT_MAX_BACKOFF_MS("cassandra.lwt_max_backoff_ms", "50"),
```

Comment:
This reduces the backoff time from 100ms to 50ms.
This doesn't seem risky but we haven't verified that on real-life workloads.

Anyway, please let PE team know about this change a the new setting